src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.h

```diff
@@ -55,6 +55,7 @@ public ref class GraphingCalculator sealed : public Windows::UI::Xaml::Data::INo
 
         void OnShowTracePopupChanged(bool newValue);
         void OnTracePointChanged(Windows::Foundation::Point newPoint);
+        void OnPointerPointChanged(Windows::Foundation::Point newPoint);
```

Comment:
>OnPointerPointChanged [](start = 13, length = 21)

This name is kind of confusing...  that said, I couldn't come up with a better name that wasn't confusing with TracePointChanged.